contracts/VoterClassVoterPool.sol

```diff
@@ -133,10 +133,18 @@ contract VoterClassVoterPool is VoterClass, ConfigurableMutable, UpgradeableCont
 
     /// @notice test if wallet represents an allowed voter for this class
     /// @return bool true if wallet is a voter
-    function isVoter(address _wallet) external view returns (bool) {
+    function isVoter(address _wallet) public view returns (bool) {
         return _voterPool[_wallet];
     }
 
+    /// @notice determine if adding a proposal is approved for this voter
+    /// @dev listed voter is required for proposal
+    /// @param _sender The address of the sender
+    /// @return bool true if this address is approved
+    function isProposalApproved(address _sender) external view returns (bool) {
+        return isVoter(_sender);
```

Comment:
proposal threshold for nouns is also dynamic based on # of nouns in circulation. 

not necessary to have for now but is there any way we can have the isApproved be based on # of tokens owned, not just "do you own a token"?

might be misunderstanding the implementation though

Reply:
Yes that is fairly easy. I will add it.   Since this is now controlled by the VoterClass communities would be free to offer individual customization and rules outside of the governance contract

Reply:
nice - lgtm